deepchem/models/torch_models/kfac_optimizer.py

```diff
@@ -332,17 +386,36 @@ def _get_matrix_form_grad(m: torch.nn.Module):
                                            -1)  # n_filters * (in_c * kw * kh)
     elif isinstance(m, torch.nn.Linear):
       p_grad_mat = m.weight.grad.data
+      try:
+        size = p_grad_mat.size()[1]
+      except IndexError:
+        p_grad_mat = p_grad_mat.unsqueeze(1)
+      print("p_grad")
+      print(p_grad_mat.size())
```

Comment:
Are the print statements necessary?

Reply:
nope will remove it, kept it there for debugging